src/db/dropFunctions.sql

```diff
@@ -0,0 +1,184 @@
+--dropFunctions.sql - Gradebook
+
+--Created by Bruno DaSilva, Andrew Figueroa, and Jonathan Middleton (Team Alpha)
+-- in support of CS305 coursework at Western Connecticut State University.
+
+--Licensed to others under CC 4.0 BY-SA-NC
+
+--This work is a derivative of Gradebook, originally developed by:
+
+--Zaid Bhujwala, Zach Boylan, Steven Rollo, Sean Murthy
+--Data Science & Systems Lab (DASSL), Western Connecticut State University (WCSU)
+
+--(C) 2017- DASSL. ALL RIGHTS RESERVED.
+--Licensed to others under CC 4.0 BY-SA-NC
+--https://creativecommons.org/licenses/by-nc-sa/4.0/
+
+--PROVIDED AS IS. NO WARRANTIES EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.
+
+--remove the following two comment lines after discussion
+--use camel case for table/field names containing more than one word
+--use hyphen in table names when combining multiple table names as in that for a m-n relationship
+
+START TRANSACTION;
+
```

Comment:
```suggestion

SET LOCAL search_path TO 'alpha', 'pg_temp';
```

Should probably set a search path to make sure functions are dropped from the correct schema